ap7.ospool.osg-htc.org guide (#181)
OSGConnect/connectbook: start/account/AP7/ap7-access.md

1. Return to the [Registration Page](https://registry.cilogon.org/registry/co_petitions/start/coef:297) and login using your institutional credentials if prompted.

1. Click your name at the top right. In the dropdown box, click "My Profile (OSG)" button.

   {: .border .border-5}
   <img src="ap7-images/ssh-homepage-dropdown.png" class= "img-fluid"/>

1. On the right hand side of your profile, click "Authenticators" link.

   {: .border .border-5 }
   <img src="ap7-images/ssh-edit-profile.png" class= "img-fluid"/>

1. On the authenticators page, click the "Manage" button.

   {: .border .border-5 }
   <img src="ap7-images/ssh-authenticator-select.png" class= "img-fluid"/>

1. On the new SSH Keys page, click "Add SSH Key" and browse your computer to upload your public SSH key.
   
   {: .border .border-5 }
   <img src="ap7-images/ssh-key-list.png" class= "img-fluid"/>

> 

Reviewer: I might add a note at the end, saying something like
> You can now log in to Access Point, `ap7.ospool.osg-htc.org` from the terminal, 
> using the same `ssh` command that was shown in the first part of the web authentication section. 
> When you log in, instead of being prompted with a web link, you should either authenticate 
> automatically or be asked for your ssh key passphrase to complete logging in.

Author: Good idea! 

Also, thank you for catching all of my terrible "login" grammer mistakes!

> 

Author: ```suggestion

You can now log in to Access Point, ap7.ospool.osg-htc.org from the terminal, using `ssh username@submit6.chtc.wisc.edu`. When you log in, instead of being prompted with a web link, you should either authenticate automatically or be asked for your ssh key passphrase to complete logging in.
```

[title]: - "OSPool ap7.ospool.osg-htc.org Access Point"
[TOC]

> *This guide is for users who were **notified by a member of the OSG team** that they will be working on the `ap7.ospool.osg-htc.org` access point.*

Author: ```suggestion
*This guide is for users who were **notified by a member of the OSG team** that they will be working on the `ap7.ospool.osg-htc.org` Access Point.*
```

The major steps to getting started with this access point are:

* applying for an OSPool `ap7.ospool.osg-htc.org` Access Point account 
* have your account approved by an OSG Team member
* logging into `ap7.ospool.osg-htc.org`

This guide will discuss how to apply for an account and login to `ap7.ospool.osg-htc.org`.

## Request Access to the OSPool `ap7.ospool.osg-htc.org` Access Point
To request access to `ap7.ospool.osg-htc.org`, submit an application using the following steps:

1. Go to the account [registration page](https://registry.cilogon.org/registry/co_petitions/start/coef:297). You will be redirected to the CILogon sign in page. Select your institution and use your institutional credentials to login. You will use these credentials later to login so it is important to remember the institution you use at this step. 
   
   {: .border .border-5 }
   <img src="ap7-images/cilogon.png" class= "img-fluid"/>
   
   
   If you have issues signing in using your institutional credentials, contact us at [support@osg-htc.org](mailto:support@osg-htc.org).

> 1. Once you sign in, you will be redirected to the "User Enrollment for OSPool ap7.ospool.osg-htc.org Access Point" page. Click "Begin" and enter your name and email address in the following page. In many cases, this information will be automatically populated. If desired, it is possible to manually edit any information automatically filled in. Once you have entered your information, click "SUBMIT".

Author: ```suggestion
2. Once you sign in, you will be redirected to the "User Enrollment for OSPool ap7.ospool.osg-htc.org Access Point" page. Click "Begin" and enter your name and email address in the following page. In many cases, this information will be automatically populated. If desired, it is possible to manually edit any information automatically filled in. Once you have entered your information, click "SUBMIT".
```

   {: .border .border-5 }
   <img src="ap7-images/comanage-enrollment-form.png" class= "img-fluid"/>

> 1. After submitting your application, you will receive an email from [registry@cilogon.org](mailto:registry@cilogon.org) to verify your email address. Click the link listed in the email to be redirected to a page confirm your invitation details. Click the "ACCEPT" button to complete this step.

Author: ```suggestion
3. After submitting your application, you will receive an email from [registry@cilogon.org](mailto:registry@cilogon.org) to verify your email address. Click the link listed in the email to be redirected to a page confirm your invitation details. Click the "ACCEPT" button to complete this step.
```

3. You will be redirected to a new page where you will be prompted to login using your institutional credentials. Once you have done so, a new page will appear with the following text: "You have successfully approved the user code. Please return to your device for further instructions."

4. Return to your terminal, and type 'Enter' to complete the login process. 

### Option 2: Log in via SSH Key Pair Authentication

It is also possible to authenticate using an SSH key pair, if you prefer. Logging in using SSH keys does not require access to an internet browser to log in into the OSG Access Point, `ap7.ospool.osg-htc.org`. 

The process below describes how to upload a public key to the registration website. It assumes that a private/public key pair has already been generated. If you need to generate a key pair, see the "Step 1: Generate SSH Keys" section of this [OSG guide](https://support.opensciencegrid.org/support/solutions/articles/12000027675-generate-ssh-keys-and-activate-your-osg-login). 

1. Return to the [Registration Page](https://registry.cilogon.org/registry/co_petitions/start/coef:297) and login using your institutional credentials if prompted.

> 1. Click your name at the top right. In the dropdown box, click "My Profile (OSG)" button.

Author: ```suggestion
2. Click your name at the top right. In the dropdown box, click "My Profile (OSG)" button.
```

   {: .border .border-5}
   <img src="ap7-images/ssh-homepage-dropdown.png" class= "img-fluid"/>

> 1. On the right hand side of your profile, click "Authenticators" link.

Author: ```suggestion
3. On the right hand side of your profile, click "Authenticators" link.
```

   {: .border .border-5 }
   <img src="ap7-images/ssh-edit-profile.png" class= "img-fluid"/>

> 1. On the authenticators page, click the "Manage" button.

Author: ```suggestion
4. On the authenticators page, click the "Manage" button.
```

   {: .border .border-5 }
   <img src="ap7-images/ssh-authenticator-select.png" class= "img-fluid"/>

> 1. On the new SSH Keys page, click "Add SSH Key" and browse your computer to upload your public SSH key.

Author: ```suggestion
5. On the new SSH Keys page, click "Add SSH Key" and browse your computer to upload your public SSH key.
```

   
   {: .border .border-5 }
   <img src="ap7-images/ssh-key-list.png" class= "img-fluid"/>

> You can now log in to Access Point, ap7.ospool.osg-htc.org from the terminal, using `ssh username@submit6.chtc.wisc.edu`. When you log in, instead of being prompted with a web link, you should either authenticate automatically or be asked for your ssh key passphrase to complete logging in.

Author: ```suggestion
You can now log in to Access Point, ap7.ospool.osg-htc.org from the terminal, using `ssh username@submit6.chtc.wisc.edu`. When you log in, instead of being prompted with a web link, you should either authenticate automatically or be asked for your ssh key passphrase to complete logging in.

```

[title]: - "OSPool ap7.ospool.osg-htc.org Access Point"
[TOC]

*This guide is for users who were **notified by a member of the OSG team** that they will be working on the `ap7.ospool.osg-htc.org` Access Point.*

The major steps to getting started with this access point are:

> * applying for an OSPool `ap7.ospool.osg-htc.org` Access Point account 
> * have your account approved by an OSG Team member
> * logging into `ap7.ospool.osg-htc.org`

Author: ```suggestion
1. Applying for an OSPool `ap7.ospool.osg-htc.org` Access Point account 
2. Have your account approved by an OSG Team member
3. Log in to `ap7.ospool.osg-htc.org`
```
